internal/api/handlers.go

```diff
@@ -360,6 +365,53 @@ func PostUserAccount(env *Environment) http.HandlerFunc {
 	}
 }
 
+func Login(env *Environment) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
```

Comment:
when sending just text, the content type should be "text/plain", or just not set.

Reply:
✅